doc/source/whatsnew/v0.24.0.rst

```diff
@@ -668,6 +669,35 @@ is the case with :attr:`Period.end_time`, for example
 
    p.end_time
 
+.. _whatsnew_0240.api_breaking.datetime_unique:
+
+The return type of :meth:`Series.unique` for datetime with timezone values has changed
+from an :class:`ndarray` of :class:`Timestamp` objects to a :class:`arrays.DatetimeArray` (:issue:`24024`).
+
+Previous Behavior:
+
+.. code-block:: ipython
+
+   In [2]: ser = pd.Series([pd.Timestamp('2000', tz='UTC'),
+      ...:                  pd.Timestamp('2000', tz='UTC')])
+   In [3]: ser.unique()
+   Out[3]: array([Timestamp('2000-01-01 00:00:00+0000', tz='UTC')], dtype=object)
+
+
+
+.. ipython:: python
+   :suppress:
+
+   ser = pd.Series([pd.Timestamp('2000', tz='UTC'),
```

Comment:
u can just show this above and remove the display from the code block

Reply:
Can you clarify?

Reply:
Got it, thanks.